bundles/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/RFXComBindingConstants.java

```diff
@@ -145,6 +145,7 @@
     private static final ThingTypeUID THING_TYPE_POWER = new ThingTypeUID(BINDING_ID, "power");
     private static final ThingTypeUID THING_TYPE_RADIATOR1 = new ThingTypeUID(BINDING_ID, "radiator1");
     private static final ThingTypeUID THING_TYPE_RAIN = new ThingTypeUID(BINDING_ID, "rain");
+    private static final ThingTypeUID THING_TYPE_RAW = new ThingTypeUID(BINDING_ID, "raw");
```

Comment:
Just out of curiosity what is the difference with `undecoded`

Reply:
UNDECODED is where the RFXCOM understands the type, but just hasn't broken it down into values. RAW is where the RFXCOM has received an RF message and hasn't tried to do anything to it at all, so the values are the ms pulse timings that would need to be processed into stable bytes before they could even be decoded.